<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceWindow</class>
 <widget class="QWidget" name="InterfaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор треугольника</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип треугольника</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="_triangleButton">
        <property name="text">
         <string>Треугольник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_isoscelesTriangleButton">
        <property name="text">
         <string>Равнобедренный треугольник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_equilateralTriangleButton">
        <property name="text">
         <string>Равносторонний треугольник</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="_labelTextLen">
       <property name="text">
        <string>Длины сторон треугольника:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="_labelA">
         <property name="text">
          <string>a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_lineEditA"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="_labelB">
         <property name="text">
          <string>b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_lineEditB"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="_labelC">
         <property name="text">
          <string>c:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_lineEditC"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="_verticalSpacerLen">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="_labelTextCoord">
       <property name="text">
        <string>Координаты начальной точки:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="_labelX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_lineEditX"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="_labelY">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="_lineEditY"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="_verticalSpacerCoord">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="_addButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
